### DISCUSSION TITLE
Rounding numbers in JSON

### DISCUSSION THREAD

**Discussion by imoknot** [fastapi/fastapi#6237] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
return  JSONResponse({"id": 9223372036854775807}) --> {  "id": 9223372036854776000 }
```

### Description

If use integer in the response is more than *** (I don't know the exact number), it is rounded up and replaces the numbers with 000
 
if id = 9223372036854775807
response
 
{
  "id": 9223372036854776000
}
 
else use 
return Response(content=json.dumps({'id':9223372036854775807})) --> {"id":9223372036854775807}
response(text) - normal
 
if JSON - abnormal
 
return  JSONResponse({"id": 9223372036854775807}) --> {  "id": 9223372036854776000 }
'id': uuid.uuid4().int >> 64
 
 
This is true when working through swagger, I did not try it through the request

### Operating System

Linux

### FastAPI Version

0.70.0

### Python Version

3.9.5

**Comment by acidjunk** [comment, marked as answer]
Not sure if this a bug:

For example, a 64-bit integer cannot be represented in JSON (since JavaScript and JSON support integers up to 2^53). Therefore, a 64-bit integer must be represented as a string in JSON requests/responses. So the type property will be set to "string", but the format property will be set to "int64" to indicate that it is a 64-bit integer.

Src: https://developers.google.com/discovery/v1/type-format
